### PR TITLE
Split out current task status into own section in the task view

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: pages/task/content/status.js [hunk 1]
```diff
@@ -95,7 +100,8 @@ module.exports = {
   },
   'with-ntco': {
     state: 'Awaiting endorsement',
-    log: 'Submitted by'
+    log: 'Submitted by',
+    currentlyWith: 'Currently with: Named Training and Competency Officer'
```
Comment (lennym) [inline pages/task/content/status.js:104]: Is this status _actually_ used for things that are with NTCO, or is it only there for b/c?

File: pages/task/content/status.js [hunk 2]
```diff
@@ -109,12 +115,17 @@ module.exports = {
...
+    currentlyWith: 'Currently with: Home Office Licensing Officer'
   },
   // TODO: content
   'awaiting-endorsement': {
     state: 'Awaiting endorsement',
-    log: 'Submitted by'
+    log: 'Submitted by',
+    currentlyWith: {
+      pil: 'Currently with: Named Training and Competency Officer',
+      project: 'Currently with: Establishment Admin'
```
Comment (lennym) [inline pages/task/content/status.js:127]: I think this answers my question.

File: pages/task/content/status.js (hunk 1 above)
Comment (wheelsandcogs) [reply pages/task/content/status.js:104]: think it's legacy.

File: pages/task/content/status.js (hunk 2 above)
Comment (wheelsandcogs) [reply pages/task/content/status.js:127]: 👍

**Pull request merged**